certbot-nginx/certbot_nginx/obj.py

```diff
@@ -96,6 +96,11 @@ def __repr__(self):
     def super_eq(self, other):
         """Check ip/port equality, with IPv6 support.
         """
+        # Check if both objects have wildcard equivalent addresses.
+        # If so, we're done.
+        any_addresses = ('0.0.0.0', '*', '')
+        if self.tup[0] in any_addresses and other.tup[0] in any_addresses:
+            return True
```

Comment:
This isn't quite right - the ports have to match as well, and the purpose of the `super` call below is that it handles ipv6 addresses. Handling ipv6 addresses might be a nice-to-have rather than necessary right now, but at minimum this method should return false for testing, say, `0.0.0.0:80` against `*:443`.